RiotSharp.AspNetCore/DistributedCache.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Extensions.Caching.Distributed;
+using System;
+using System.Collections.Generic;
+
+namespace RiotSharp.AspNetCore
+{
+    /// <summary>
+    /// Implementation of ICache with AspNetCore's distributed cache
+    /// </summary>
+    public class DistributedCache : ICache
+    {
+        private IDistributedCache distributed;
+        private List<object> usedKeys;
+
+#pragma warning disable CS1591
+        public DistributedCache(IDistributedCache memoryCache)
+        {
+            this.distributed = memoryCache;
+            usedKeys = new List<object>();
+        }
+
+        public void Add<K, V>(K key, V value, TimeSpan slidingExpiry) where V : class
```

Comment:
could we add expliit overrides, for the hybrid cache too?

Reply:
What do you mean by explicit overrides?

Reply:
`public override void Add...`

Reply:
I can't override it, because it's not inherited from a virtual method. Furthermore, if override it with the `new` keyword, the method won't fulfill the requirements of the interface.